examples/yagna-config-template.yaml

```diff
@@ -0,0 +1,7 @@
+yagna:
```

Comment:
That's because I missed the fact we have separate directories for `examples` and `configs`. I think I assumed both file types would live inside `examples` (perhaps with `configs` moved inside `examples`). Maybe that would make more sense?

Reply:
okay, let's discuss it... I prefer to have configs separate from the dapp descriptor examples

Reply:
After a discussion I'm okay with keeping the current directory structure. A better approach would be to have a proper readme and/or a command which generates a default config.

Reply:
@zakaprov are you going to remove this config file then?